From leoli@freescale.com  Thu Aug 13 16:44:31 2009
From: Li Yang <leoli@freescale.com>
Date: Tue, 11 Aug 2009 11:11:11 +0800
Subject: USB: gadget: Update Freescale UDC entry in MAINTAINERS
To: gregkh@suse.de, joe@perches.com
Cc: lg@denx.de, linux-usb@vger.kernel.org, linuxppc-dev@ozlabs.org, Li Yang <leoli@freescale.com>
Message-ID: <1249960271-3400-1-git-send-email-leoli@freescale.com>


Change the F entry for file rename, and make it also cover fsl_qe_udc
driver.  Update the name accordingly.

Signed-off-by: Li Yang <leoli@freescale.com>
Acked-by: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Cc: Joe Perches <joe@perches.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 MAINTAINERS |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -2107,12 +2107,12 @@ S:	Supported
 F:	arch/powerpc/sysdev/qe_lib/
 F:	arch/powerpc/include/asm/*qe.h
 
-FREESCALE HIGHSPEED USB DEVICE DRIVER
+FREESCALE USB PERIPHERIAL DRIVERS
 M:	Li Yang <leoli@freescale.com>
 L:	linux-usb@vger.kernel.org
 L:	linuxppc-dev@ozlabs.org
 S:	Maintained
-F:	drivers/usb/gadget/fsl_usb2_udc.c
+F:	drivers/usb/gadget/fsl*
 
 FREESCALE QUICC ENGINE UCC ETHERNET DRIVER
 M:	Li Yang <leoli@freescale.com>
